src/PrestaShopBundle/Command/SecurityAttributeLinterCommand.php

```diff
@@ -37,11 +37,11 @@
 use Symfony\Component\Routing\Route;
 
 /**
- * Checks if all admin routes have @AdminSecurity configured
+ * Checks if all admin routes have #[AdminSecurity] configured
  *
  * @see \PrestaShopBundle\Security\Annotation\AdminSecurity
  */
-final class SecurityAnnotationLinterCommand extends Command
+final class SecurityAttributeLinterCommand extends Command
```

Comment:
The order must also be updated. I did not go as far as managing attributes AND annotations

Reply:
Attributes alone is enough we won't use annotation in the core anymore